README.md

```diff
@@ -21,37 +21,145 @@ would yield a response similar to:
 
     {"total":1,"items":[{"id":9560,"views":227913,"downloads":16498,"crossref":103,"pubmed":21,"scopus":52}]}
 
-## installation
+------------
 
-[code](https://github.com/elifesciences/elife-metrics/blob/master/install.sh) 
+## Local development
+
+### Prerequisites
+
+Docker
+Docker Compose
+
+#### 3rd Party credentials for local development
+
+The service has dependencies on the following 3rd party services:  
+
+[`crossref`](https://www.crossref.org)
+[`scopus`](https://www.scopus.com/home.uri)
+[`Google Analytics`](https://developers.google.com/analytics/devguides/collection/ga4)
+
+Each of these require credentials to be set in the environment. You will need to set these up in your local environment 
+in order to ingest/generate metrics data locally.
+
+In the `.docker/app.cfg` file, you will need to set the following variables with real values:
+
+```
+[scopus]
+apikey: <scopus api key>
+
+[crossref]
+user: <crossref user>
+pass: <crossref pass>
+```
+For Google Analytics, you will need to provide a `client_secrets.json` file in the root of the project.
+
+Example `client-secrets.json` file:
+```json
+{
+  "private_key_id": "<private_key_id>",
+  "private_key": "<private_key>",
+  "client_id": "<client_id>",
+  "client_email": "<client_email>",
+  "type": "service_account"
+}
+```
+
+#### TODO: AWS Section
+...
+
+### (Optional) Prepare for seeding the local database
+If you want to seed the local database with some data, one way is to populate the `.docker/pg_import_data.sql` file with the
+ desired contents. In a later step you can execute the import command to load the data into the database.
+
+### Build
+```bash
+docker-compose build
```

Comment:
This is more a question. On another repo, I started to get issues with `docker-compose`. Whereas `docker compose` worked and seem to be the newer version. Just wondering whether we should move to that.

Reply:
I will take a look at this 👍🏻

Reply:
Both will work `docker-compose` (external tool) and `docker compose` (Docker sub-command), but I have updated the README to use the sub-command 👍🏻